### PR TITLE
Add a keyPrefix option

### PR DESCRIPTION
**Pull request by Archanium**
### Background
I've got a need for the keyPrefix mentioned in #73, so this is a little attempt at adding that. :) 

### Proposed changes
Adding a keyPrefix option, so that it's possibly to upload files to a prefixed s3 path.

### Proposed reviewers (optional)

[@ mentions of other contributors]

### CODE REVIEW

**Comment by fernando-mc** [comment]
@Archanium this looks pretty cool. Can you do me a favor and rebase and test it again? Then I can bring it into a new version.

File: index.js [hunk 1]
```diff
@@ -76,24 +78,32 @@ class Client {
...
+
                   if (manageResources === false) {
                     this.serverless.cli.log(`Success! Your files have been removed`);
                   } else {
-                    this.serverless.cli.log(
-                      `Success! Your files have been removed and your bucket has been deleted`
-                    );
+                    if (!keyPrefix) {
+                      this.serverless.cli.log(
+                        `Success! Your files have been removed and your bucket has been deleted`
```
Comment (fernando-mc) [inline index.js]: Can you add a logging statement here for if(keyPrefix) 

Let's give the user clarification that files from the keyPrefix are deleted.

**Comment by fernando-mc** [comment]
@Archanium Actually, don't worry about it. I should have asked you to do that before. I'll merge now and add that myself with the new release.

**Comment by fernando-mc** [comment]
Just pushed this. Should be out soon. Had to fix an issue that came up when keyPrefix was undefined but seems to work now. Thanks again!

**Comment by Archanium** [comment]
You're very welcome! :) Thanks for the rest of it!

**Pull request merged**